app/src/services/census/CensusStreamService.ts

```diff
@@ -103,4 +119,26 @@ export default class CensusStreamService implements ServiceInterface {
             }
         });
     }
+
+    private startMessageTimer(): void {
+        CensusStreamService.logger.info('Census message timer started');
+
+        this.messageTimer = setInterval(() => {
+            CensusStreamService.logger.debug('Census message timeout check running...');
+
+            this.lastMessagesMap.forEach((lastTime: number, world: World) => {
+                const thresholdLimit = 60000;
+                const threshold: number = Date.now() - thresholdLimit; // We expect to get at least one death event on every world, regarless of time within 60 seconds
+
+                if (!lastTime) {
```

Comment:
If the state is not possible to reach, then isn't this kind of redundant?

Reply:
Indeed